pymatgen/core/surface.py

```diff
@@ -1104,10 +1103,8 @@ def get_slab(self, shift: float = 0, tol: float = 0.1, energy: float | None = No
         species = self.oriented_unit_cell.species_and_occu
 
         # Shift all atoms
-        # DEBUG(@DanielYang59): shift value in Angstrom inconsistent with frac_coordis
         frac_coords = self.oriented_unit_cell.frac_coords
-        # DEBUG(@DanielYang59): suspicious shift direction (positive for downwards shift)
-        frac_coords = np.array(frac_coords) + np.array([0, 0, -shift])[None, :]
+        frac_coords = np.array(frac_coords) + np.array([0, 0, shift])[None, :]
```

Comment:
https://github.com/materialsproject/pymatgen/pull/3691#discussion_r1553711017 migrated here. Clarified two issues:
- The unit for `shift` should be "fractional" instead of "Angstrom", docstring has been updated.
- (Breaking) Reversed the default shifting direction, previously a positive shift vector refers to shift "downwards".

Reply:
if we change the shifting direction, that needs to be very prominently communicated in the changelog and the [recent breakage](https://pymatgen.org/compatibility.html#recent-breaking-changes) docs. could there be some intuition we're not thinking of why positive shift should decrease the z-axis? just trying to weigh the pros and cons of breaking. i also would expected z to increase but maybe explaining in the doc string why it's this way could warn users and avoid breaking.

Reply:
> if we change the shifting direction, that needs to be very prominently communicated in the changelog and the [recent breakage](https://pymatgen.org/compatibility.html#recent-breaking-changes) docs. 

Okay! I would keep it in mind.

> could there be some intuition we're not thinking of why positive shift should decrease the z-axis? just trying to weigh the pros and cons of breaking. i also would expected z to increase but maybe explaining in the doc string why it's this way could warn users and avoid breaking.

I read through the entire module earlier and didn't notice much good reason for this. Also this counter-intuitive behavior is not explained in docstring/comment either. Tried to ping the original author (@CifLord I believe) but no reply yet.